Sub/Src/ComputerVision/vision.py

```diff
@@ -118,15 +122,32 @@ def run(self):
                     #Perform solve pnp calculations
                     self.distance_calculator.set_coordinates(r, i, x, y, w, h)
                     rotation, translation, distance = self.distance_calculator.calculate_distance()
-                    
-                    print('Rotation: ', rotation)
-                    print('Translation: ', translation)
-                    print('Distance: ', distance)
+                    #print(distance)
                     xmin, ymin, xmax, ymax = convertBack(float(x), float(y), float(w), float(h))
                     pt1 = (xmin, ymin)
                     pt2 = (xmax, ymax)
                     cv2.rectangle(byte_frame, pt1, pt2, (0, 255, 0), 2)
                     cv2.putText(byte_frame, i[0].decode() + " [" + str(round(i[1] * 100, 2)) + "]", (pt1[0], pt1[1] + 20), cv2.FONT_HERSHEY_SIMPLEX, 1, [0, 255, 0], 4)
+                    cv2.putText(byte_frame, "[" + str(round(distance, 2)) + "ft]", (pt2[0], pt1[1] + 40), cv2.FONT_HERSHEY_SIMPLEX, 1, [0,127, 127], 4)
+                    label = i[0].decode("utf-8")
+                    detection_data = struct.pack('sfffffffffff',
+                                                 label.encode("utf-8"),
+                                                 i[1],
+                                                 i[2][0],
+                                                 i[2][1],
+                                                 i[2][2],
+                                                 i[2][3],
+                                                 rotation[0],
+                                                 rotation[1],
+                                                 rotation[2],
+                                                 translation[0],
+                                                 translation[1],
+                                                 translation[2])
+
+                    if ((time.time() - start_time) >= 0.5):
```

Comment:
Make parameters like 0.5 be something that can be easily changed (maybe the parameter server)

Reply:
Done!